src/Card/index.jsx

```diff
@@ -0,0 +1,5 @@
+export { default } from 'react-bootstrap/Card';
```

Comment:
Curious if we should just leave the react-bootstrap export structure as-is rather than bundling them/changing where you import them from.  Is it that this more closely matches what we're doing elsewhere in Paragon?

Reply:
Today, I'm finding that if we need to modify or replace the react-bootstrap components that I have to provide very detailed instructions on how to do it since it becomes, "make a directory, import the react-bootstrap, component make your shim, rexport it.

I think it's clearer if we make components uniformly have their own index.jsx in Paragon.

Reply:
Agree x 100